global.json

```diff
@@ -1,6 +1,6 @@
 {
     "projects": [ "src" ],
     "sdk": {
-        "version": "1.0.0-preview2-003131"
+        "version": "1.0.3"
```

Comment:
The bootstrapper build scripts will need updating to match this - they specify a version of core to install. (The cake version number in the same place is also out of date, if you wouldn't mind, while you're there! 😄 )